packages/variable-editor/src/components/variables/data/test-utils/types.ts

```diff
@@ -0,0 +1,60 @@
+import { type Cell, type FilterMeta, type Row, type RowPinningPosition } from '@tanstack/react-table';
+
+export class MockRow<TData> implements Row<TData> {
```

Comment:
Maybe someone with more typescript knowledge than I have is able to simplify this mock definition, so that we only need to mock out the parts we're actually programming against?

Reply:
exactly! not ideal, otherwise if we update tanstack table and a new property gets added this will fail.

Also most of the times classes should not be used in js just use types and objects.
if you need a creator then do a creator function which returns a row object.
e.g. like here: https://github.com/axonivy/inscription-client/blob/master/packages/editor/src/components/browser/function/parent-name.test.ts